sky/utils/command_runner.py

```diff
@@ -81,6 +82,14 @@ def ssh_options_list(ssh_private_key: Optional[str],
         '-i',
         ssh_private_key,
     ] if ssh_private_key is not None else []
+
+    if ssh_proxy_command is not None:
+        logger.info(f'--- Proxy: {ssh_proxy_command} ---')
```

Comment:
Do we want to make it `logger.debug` before merging?

Reply:
Done.